exe/floe

```diff
@@ -7,14 +7,15 @@ require "optimist"
 opts = Optimist.options do
   version "v#{Floe::VERSION}\n"
   opt :workflow, "Path to your workflow json", :type => :string, :required => true
-  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => '{}'
```

Comment:
Not really related to this change but we changed this to `input` and `output` in the states, this should stay consistent